tests/shutdown/shutdown.pm

```diff
@@ -31,6 +31,8 @@ sub run {
     if (!sle_version_at_least('12-SP2') && check_var('VIRTIO_CONSOLE', 1)) {
         add_serial_console('hvc0');
     }
+    # Clean dhcp ids from hdd image
+    assert_script_run("rm /var/lib/wicked/*.xml");
```

Comment:
Not sure if there are maybe other important files in that directory. However we should at least expand it with the force parameter (`-f`) to avoid eventual prompts (e.g. zsh does this). `-v` maybe also makes sense to have a more traceable result what was deleted (useful for debugging).

Reply:
@thehejik @nicksinger network + wickedd stop + ls + rm -f => http://10.100.12.105/tests/931#step/shutdown/9